src/js/event-target.js

```diff
@@ -23,7 +23,12 @@ EventTarget.prototype.off = function(type, fn) {
 EventTarget.prototype.removeEventListener = EventTarget.prototype.off;
 
 EventTarget.prototype.one = function(type, fn) {
+  // Remove the addEventListener alias before calling Events.on
+  // so we don't get into an infinite type loop
+  let ael = this.addEventListener;
+  this.addEventListener = () => {};
```

Comment:
I think it works fine because the event system creates a clone of the handlers before it starts triggering events: https://github.com/videojs/video.js/blob/2e2dbde4b4b70ecac22736477ca60d20a12cfadd/src/js/utils/events.js#L52-L63